sass/mixins/_breadcrumb.scss

```diff
@@ -13,6 +13,8 @@
   @include linkButton();
   font-size: $font-size-smaller;
   margin-right: 15px;
+  outline: 1px solid transparent;
```

Comment:
Looking at this PR and JSUI-3079, I think it be useful to extract this line as a mixin. If we ever need to adjust the outline, we can do it in one place.